build.sbt

```diff
@@ -14,30 +14,45 @@
  * limitations under the License.
  */
 
+/***************************
+* Spark Packages settings *
```

Comment:
Can you move this below similar to https://github.com/delta-io/delta/blob/master/build.sbt?

Reply:
Moved thanks.